Google.Api.Gax.Grpc.Tests/Rest/HttpRulePathPatternTest.cs

```diff
@@ -18,7 +18,10 @@ public class HttpRulePathPatternTest
             { "x/y:custom", new RuleTestRequest(), "x/y:custom" },
             { "firstPart/{x}/secondPart/{y}", new RuleTestRequest { X = "x1", Y = "y2" }, "firstPart/x1/secondPart/y2" },
             { "combined/{x}-{y}/end", new RuleTestRequest { X = "xx", Y = "yy" }, "combined/xx-yy/end" },
+            { "pattern/{x}", new RuleTestRequest { X = "abc/def" }, "pattern/abc%2Fdef" },
             { "pattern/{x=abc/*}", new RuleTestRequest { X = "abc/def" }, "pattern/abc/def" },
+            { "pattern/{x=abc/*}", new RuleTestRequest { X = "abc/San Jose" }, "pattern/abc/San%20Jose" },
```

Comment:
Possibly use San Francisco or New York as an example instead of San Jose? I had to look up whether San Jose had an accent or not usually... and given that we're talking about escaping, I was expecting it to *be* an example of escaping accents.

Possibly add another example that *does* have an accent, e.g. "abc/caf\u00e9"? That would help to validate that we're escaping using UTF-8.

Reply:
It was `San José` initially but the `é` gets mangled somewhere, and in the CI it is shown as a replacement character (`�`), so the test no longer works. I decided against trying to figure out where the problem happens at the moment.
Changing to New York.

Reply:
Right, that's why I was suggesting `caf\u00e9` to make the source code fully ASCII, but still representing a non-ASCII value. I still think it would be good to do that.

Reply:
Thank you, implemented.